supervisor/errata_utils.py

```diff
@@ -157,6 +157,38 @@ def get_erratum_for_link(link: str, full: bool = False) -> Erratum | FullErratum
     return get_erratum(erratum_id, full=full)
 
 
+class ErratumBuilds(BaseModel):
+    product_list: dict[str, dict[str, set[str]]]
+
+    def __eq__(self, other: object) -> bool:
```

Comment:
This shouldn't be needed for pydantic, whether using BaseModel or RootModel - the default is field-by-field comparison.